flyteadmin/pkg/manager/impl/validation/signal_validator.go

```diff
@@ -76,6 +77,22 @@ func ValidateSignalSetRequest(ctx context.Context, db repositoryInterfaces.Repos
 	if err != nil {
 		return err
 	}
+	err = propellervalidators.ValidateLiteralType(valueType)
+	if err != nil {
+		return errors.NewFlyteAdminErrorf(codes.InvalidArgument,
```

Comment:
Could we add a new error like [this](https://github.com/flyteorg/flyte/blob/bed761c33c40af23750467c828afea553c0b80a0/flyteadmin/pkg/errors/errors.go#L130) and return the new one here?

Reply:
done, thank you